build.gradle.kts

```diff
@@ -112,14 +114,6 @@ tasks.shadowJar {
         into("META-INF/services/")
     }
 
-    doFirst {
```

Comment:
What's the purpose of removing this code block? That's about service provider support.

Reply:
We can include the `META-INF/services/java.sql.Driver` file directly in the project, i.e. `src/main/resources/META-INF/services/java.sql.Driver`, this allows for:
1. Avoids the need to call `Class.forName(software.aws.rds.jdbc.mysql.Driver)` when writing tests utilizing `DriverManager.getConnection()`
2. Removes the need to create a new file every time we build the JAR